### PR TITLE
Fixed Assassin List and targets

### PR DESCRIPTION
**Pull request by orefifa12**
- Whenever a player is killed, the targets and assassins update
- The killed player's assassins updates to empty after kill

### CODE REVIEW

File: src/components/KillButton.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { query, updateDoc, collection, getDocs, where } from "firebase/firestore
 const KillButton = (props) => {
     const [selectedPlayer, setSelectedPlayer] = useState('');
     const [showAlert, setShowAlert] = useState(false); // State to control the alert visibility
+    const [editableArray, setEditableArray] = useState([]);
```
Comment (Octophi) [inline src/components/KillButton.js]: Q: Does this editableArray need to be a state variable, and if so, where is it getting used? I might just be missing something here

Comment (orefifa12) [reply src/components/KillButton.js]: I removed them in the most recent commit. I may have used them initially in an earlier commit I think, but not in use anymore.

File: src/components/AssasinsSelection.js [hunk 2]
```diff
@@ -47,9 +46,8 @@ const AssasinsSelection = (props) => {
             const index = targetArray.indexOf(killedPlayerNamed);
             const spliced = targetArray.slice();
             spliced.splice(index, 1);
```
Comment (Octophi) [inline src/components/AssasinsSelection.js]: Is spliced getting used anymore?

Comment (orefifa12) [reply src/components/AssasinsSelection.js]: Yeah, it wasn't. I think there should be no more extra lines.

**Review by Octophi — APPROVED** [review]

**Pull request merged**